azure-pipelines.yml

```diff
@@ -0,0 +1,47 @@
+# Build Docker image for this app using Azure Pipelines
+# http://docs.microsoft.com/azure/devops/pipelines/languages/docker?view=vsts
+pool:
+  vmImage: 'Ubuntu 16.04'
+
+variables:
+  imageName: 'ioet/bpm-projects-api:latest'
+
+steps:
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: 3.6
+    architecture: 'x64'
+
+
+- task: PythonScript@0
+  displayName: 'Exp'
+
+- script: |
+    RUN apt-get update
+    RUN apt-get install -y git
+    RUN apt-get install -y python3 python3-dev python3-pip
+    RUN apt-get install -y curl
+  displayName: 'Install OS requirements'
+
+- script: |
+    pip3 install -r requirements.txt
+  displayName: 'Install requirements'
+
+- script: |
+    pip3 install pytest
+    pip3 install pytest-cov
+    pytest tests --doctest-modules --junitxml=junit/test-results.xml --cov=com --cov-report=xml --cov-report=html
```

Comment:
I don't know if flake8 also runs over .yml files, but this line seems a bit too long for me

Reply:
I have not seen a way to break a line of a command in Azure pipelines yml configurations, which are not only yml files, but a DSL and you cannot break the line as if it was a yml `string`. Been that said its tolerable. Not that much for real programming languages like Python where the recommended max length is `80` and there are real ways to break those lines.